scripts/docker/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2024 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Shell script for building Cloud Orchestrator
+
+dir=$(dirname $0)
+pushd $dir > /dev/null
```

Comment:
same as another comment, I am wondering if we really need this.

Reply:
ditto.